app/controllers/api/v1/search_results_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SearchResultsController < ApplicationController
```

Comment:
🎨 Should we name it like this? Because the search-results, feel like it is a search-result listing. While here we are searching the keywords which has URL contains a word (search term).

```suggestion
    class SearchKeywordsController < ApplicationController
```

Reply:
solved in a9fa777